clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -1057,17 +1055,17 @@ class ArchiveFileHandler final : public FileHandler {
             if (Error Err = OFH.ReadBundle(OS, *Buf))
               return Err;
           } else if (Mode == OutputType::Archive) {
-            auto &Name =
-                ArNames.emplace_back((TT + "." + *ChildNameOrErr).str());
-            auto &Data = ArData.emplace_back();
+            // Extract the bundle to a buffer.
+            SmallVector<char, 0u> Data;
```

Comment:
NIT:
Won't default '0' will always lead to re-allocation? AFAICS, docs recommend omit N in template args if the choice can't be reasonably made.
```suggestion
            SmallVector<char> Data;
```